docs/examples/general/expressions/expr_type_promotions.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p><code></code><code>   This simplifies the manipulation of types. In an actual scenario the data that is processed by the DALI arithmetic operators will be tensors that are produced by another operator that contains some          images, video sequences, or other data.</code><code></code></p><p>Unnecessary spaces.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>